src/wled/models.py

```diff
@@ -130,7 +131,7 @@ class Segment:
     stop: int
 
     @staticmethod
-    def from_dict(
+    def from_dict(  # pylint: disable=R0914
```

Comment:
![image](https://user-images.githubusercontent.com/6491743/228315474-709cde24-3384-47bc-8ec7-11d20753ef54.png)

I was getting a `pylint` error so turned this off...